src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -109,9 +109,9 @@ public void handle(CellEditEvent<TransactionModel, String> t) {
 
                 updateTransactionTableView();
             } catch (StorageException e) {
-                System.out.println("Error editing transaction date: " + e.getMessage());
+                setupErrorPopup("Error editing transaction date.");
             } catch (ParseException e) {
-                System.out.println("Error parsing date string: " + e.getMessage());
+                setupErrorPopup("Error parsing date string.");
```

Comment:
Should probably keep the message. Latter definitely want to translate it but even the bare message should be useful.